docs/api/toolbar.md

```diff
@@ -24,941 +17,76 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
-## Buttons
+import Basic from '@site/static/usage/toolbar/basic/index.md';
 
-Buttons placed in a toolbar should be placed inside of the `<ion-buttons>` element. The `<ion-buttons>` element can be positioned inside of the toolbar using a named slot. The below chart has a description of each slot.
+<Basic />
 
-| Slot         | Description                                                                                              |
-|--------------|----------------------------------------------------------------------------------------------------------|
-| `secondary`  | Positions element to the `left` of the content in `ios` mode, and directly to the `right` in `md` mode.  |
-| `primary`    | Positions element to the `right` of the content in `ios` mode, and to the far `right` in `md` mode.      |
-| `start`      | Positions to the `left` of the content in LTR, and to the `right` in RTL.                                |
-| `end`        | Positions to the `right` of the content in LTR, and to the `left` in RTL.                                |
 
+## Buttons in Toolbars
 
-## Borders
+Buttons placed in a toolbar should be placed inside of the [buttons](./buttons) component. The buttons component can be positioned inside of the toolbar using a named [slot](#slots). The `"primary"` and `"secondary"` slots behave differently in `ios` and `md` mode. The buttons component can wrap a standard [button](./button), [back button](./back-button), a [menu button](./menu-button), or several of any of them. A button in a toolbar is styled to be clear by default, but this can be changed using the [`fill`](./button#fill) property on the button. Back buttons are linked to the app's navigation and will only show up when there is a previous page in the navigation stack, but we can bypass this by setting the `defaultHref` property. The menu button is set to only show when a linked menu is active, but this can be overridden by setting the `autoHide` property to `false`.
 
-In `md` mode, the `<ion-header>` will receive a box-shadow on the bottom, and the `<ion-footer>` will receive a box-shadow on the top.  In `ios` mode, the `<ion-header>` will receive a border on the bottom, and the `<ion-footer>` will receive a border on the top.
+import Buttons from '@site/static/usage/toolbar/buttons/index.md';
+
+<Buttons />
+
+
+## Searchbars in Toolbars
+
+A [searchbar](./searchbar) can be placed inside of a toolbar to search through the content. It should be the only child component of the toolbar, and will take up the full width and height.
+
+import Searchbars from '@site/static/usage/toolbar/searchbars/index.md';
```

Comment:
Would it make sense to import the toolbar playground from `ion-searchbar` instead? (Or have searchbar import this one.) The two are more or less identical, so that would cut down on code we're maintaining. I could see it being confusing though, so I'm open to keeping both.

Ditto for the segments playground below, though you may need to merge the latest `main` into this branch first to grab those.

Reply:
Oh that's a good idea. I'm fine with sharing them unless we think it will be confusing to maintain. Maybe a code comment above the import is sufficient to avoid confusion?